frontend/server/src/Controllers/Problem.php

```diff
@@ -5019,6 +5019,9 @@ public static function getProblemNewForSmarty(
                                 'source' => strval($r['source']),
                                 'visibility' => intval($r['visibility']),
                                 'statusError' => $statusError,
+                                'parameter' => !empty($response['parameter'])
+                                  ? strval($response['parameter'])
+                                  : '',
```

Comment:
```suggestion
                                  : null,
```

es mejor usar `null` para indicar que algo falta en vez de "valores mágicos".

Reply:
Listo